src/senaite/jsonapi/api.py

```diff
@@ -271,6 +271,12 @@ def get_info(brain_or_object, endpoint=None, complete=False):
             return {}
         complete = True
 
+    # When querying uid catalog we have to be sure that we skip the objects
+    # used to relate two or more objects
+    if api.is_relationship_object(brain_or_object):
+        logger.warn("Skipping relationship object {}".format(brain_or_object))
```

Comment:
passing in objects into format is kind of bad habit. Better use `.format(repr(brain_or_object)))`